### ISSUE TITLE
Run ontrack.jar failed

### ISSUE THREAD

**Issue by flesire** [nemerosa/ontrack#298]
When i try to run latest version of ontrack (2.13.8) i have the following error :

***Error: Invalid or corrupt jarfile ontrack.jar***

**Comment by dcoraboeuf** [comment]
Interesting... I have no access now, but I'll have a look soon.

I've refactored the complete delivery chain for Ontrack (see #289). This might be a side effect.

However, when I go to https://ontrack.nemerosa.net, where 2.13.8 is deployed, there is no issue. So this might be a pure GitHub upload issue.

**Comment by dcoraboeuf** [comment]
Did you download the JAR from GitHub, or did you build it yourself? And if yes, how?

**Comment by flesire** [comment]
I've download from github via wget and manually on linux and windows and i always have the problem.

**Comment by dcoraboeuf** [comment]
I'll have a look tonight.

**Comment by dcoraboeuf** [comment]
I confirm the problem:

`java -jar ontrack.jar` raises the error

but:

* `jar tf ontrack.jar`
* `unzip ontrack.jar`

do not fail...

**Comment by dcoraboeuf** [comment]
I confirm that the JAR uploaded in GitHub is not correct, when compared to the ones created by the Jenkins pipeline:

```
-rw-r-----@  1 damien  staff  49697979 Jul 25 10:44 ontrack-2.13.8-github.jar
-rw-r-----@  1 damien  staff  49697733 Jul 25 10:45 ontrack-2.13.8-jenkins-build.jar
-rw-r-----@  1 damien  staff  49697733 Jul 25 10:45 ontrack-2.13.8-jenkins-publish.jar
```

I'll check for the previous releases.

**Comment by dcoraboeuf** [comment]
Same thing for another previous release:

```
-rw-r-----@  1 damien  staff  49694506 Jul 25 10:59 ontrack-2.13.4-github.jar
-rw-r-----@  1 damien  staff  49694248 Jul 25 11:00 ontrack-2.13.4-jenkins-build.jar
```

The version uploaded to GitHub is always slightly bigger.

**Comment by dcoraboeuf** [comment]
The release script uploads the Jar using multipart http body when it should be a binary content, with the `application/zip` content type. Opening nemerosa/http-client#4 to have access to such a method.

**Comment by dcoraboeuf** [comment]
Fixed in 2.13.9
